tests/e2e/specs/shopper/active-filters.test.ts

```diff
@@ -66,7 +66,7 @@ const insertBlocks = async () => {
 	await insertBlock( block.name );
 };
 
-const configurateFilterProductsByAttributeBlock = async (
+const configurateFilterByAttributeBlock = async (
```

Comment:
I think I'm being very picky here, but what about `configure` instead of `configurate`? I think it's more appropriate in this case. Obviously it's really minor suggestion and doesn't serve the goal of this PR, feel free to ignore 😅 

```suggestion
const configureFilterByAttributeBlock = async (
```